pkg/plugins/golang/v2/api.go

```diff
@@ -143,7 +143,7 @@ func (p *createAPISubcommand) Validate() error {
 	// In case we want to scaffold a resource API we need to do some checks
 	if p.doResource {
 		// Check that resource doesn't exist or flag force was set
-		if !p.force && p.config.HasResource(p.resource.GVK()) {
+		if !p.force && p.config.HasResource(p.resource.ResourceData()) {
```

Comment:
If you apply the above change, this would be:
```suggestion
		if !p.force && p.config.HasResource(p.resource.Data()) {
```
which IMO is easier to read.

(There are other places too with similar statements, won't mark them all to avoid noise.)